core/lib/presentation/views/text/text_form_field_builder.dart

```diff
@@ -75,7 +75,7 @@ class _TextFieldFormBuilderState extends State<TextFormFieldBuilder> {
 
   @override
   Widget build(BuildContext context) {
-    return TextFormField(
+    return TextField(
```

Comment:
why we need to change this

Reply:
Since we don't need to use auto validate functions, we only need `TextField`. Avoid leading to unnecessary errors